docs/sources/send-data/otel/native_otlp_vs_loki_exporter.md

```diff
@@ -0,0 +1,107 @@
+---
+title: How is native OTLP endpoint different from Loki Exporter
+menuTitle:  Native OTLP endpoint vs Loki Exporter
+description: Native OTLP endpoint vs Loki Exporter
+weight:  251
+---
+
+# How is native OTLP endpoint different from Loki Exporter
+
+## Introduction
+
+OpenTelemetry (OTel) is quickly becoming an industry standard with increasing adoption. Prior to the Loki 3.0 release, there was no native support for ingesting OTel logs to Loki, which led to creation of the [LokiExporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/README.md). While the LokiExporter got the job done of ingesting OTel logs to Loki, it did not provide a native user experience, and the querying experience was not optimal. As part of our effort to improve user experience with OTel, we added native OTel log ingestion support to Loki with 3.0 release.
+
+## What has changed?
+
+### Formatting of logs
+
+[LokiExporter's README](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/lokiexporter/README.md) explains in detail the formatting of OTel logs ingested via LokiExporter, while Loki’s native OTel log ingestion endpoint is described in detail in its [documentation](https://grafana.com/docs/loki/<LOKI_VERSION>/send-data/otel/). Here is a summary of how the logs are formatted between the two:
+
+**LokiExporter**
+
+- Index Labels:
+  - It supports label control via hints set by the OTel client/collector.
+  - Default labels:
+    - job=service.namespace/service.name
+    - instance=service.instance.id
+    - exporter=OTLP
+    - level=severity
+- Log Body: Encodes log records and attributes in json(default) or logfmt format.
+
+**Loki’s native OTel log ingestion endpoint**
+
+- Index Labels:
+  - It supports label control via per-tenant OTLP configuration in Loki.
+  - By default, it picks some pre-configured resource attributes as index labels as explained [here](http://localhost:3002/docs/loki/<LOKI_VERSION>/send-data/otel/#format-considerations).
```

Comment:
`http://localhost:3002/` this links needs correction

Reply:
Thanks for catching it. I have fixed the link.